kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -709,31 +725,111 @@ public Properties getOriginalProperties() {
     return properties;
   }
 
+  public final Map<String, Object> getSchemaRegistryConfigs() {
+    HashMap<String, Object> possibleConfigs = new HashMap<>();
+
+    // Use a default schema.registry.url, if not specified.
+    possibleConfigs.put(SCHEMA_REGISTRY_URL_CONFIG, SCHEMA_REGISTRY_URL_DEFAULT);
+
+    possibleConfigs.putAll(originals());
+    possibleConfigs.putAll(originalsWithPrefix("schema.registry.", /* strip= */ true));
+    possibleConfigs.putAll(originalsWithPrefix("client.", /* strip= */ true));
+    possibleConfigs.putAll(originalsWithPrefix("producer.", /* strip= */ true));
+    possibleConfigs.putAll(originalsWithPrefix("consumer.", /* strip= */ true));
+
+    // Disable auto-registration of schemas. Schema registration is handled by SchemaManager.
+    possibleConfigs.put(AUTO_REGISTER_SCHEMAS, false);
+    // Disable latest-version fetching of schemas.
+    possibleConfigs.put(USE_LATEST_VERSION, false);
+
+    return Maps.filterKeys(
+        possibleConfigs, AbstractKafkaSchemaSerDeConfig.baseConfigDef().names()::contains);
+  }
+
+  public final Map<String, Object> getJsonSerializerConfigs() {
+    HashMap<String, Object> serializerConfigs = new HashMap<>();
+    serializerConfigs.putAll(originalsWithPrefix("client.", /* strip= */ true));
+    serializerConfigs.putAll(originalsWithPrefix("producer.", /* strip= */ true));
```

Comment:
the first three lines could be factored out and reused, but whatever, marginal, if better.

Reply:
Done.